.github/workflows/labeler.yml

```diff
@@ -9,4 +9,4 @@ jobs:
     - uses: actions/labeler@main
       with:
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
-        sync-labels: true
+        sync-labels: false
```

Comment:
I was reading the definition of `sync-labels` which says "Whether or not to remove labels when matching files are reverted or no longer changed by the PR" I wonder if we still want this to be `true`.

Reply:
I think this is what was removing the label that we had manually added. I think we don't want the bots to have this power.

Reply:
That makes sense. Thanks for fixing this 🎉